scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -24,7 +24,9 @@ workflow GvsImportGenomes {
   if ((input_length != length(external_sample_names)) || (input_indexes_length != length(external_sample_names))) {
     call TerminateWorkflow {
       input:
-        message = "The number of external_sample_names, sample input_vcfs and sample input_vcf_indexes are not the same."
+        external_sample_count = length(external_sample_names),
+        input_length = input_length,
+        input_indexes_length = input_indexes_length
```

Comment:
do we use TerminateWorkflow anywhere else? We're making it very specific here---does it just echo the message?!?!

Reply:
Yeah the `TerminateWorkflow` name seems very generic but actually it's only ever called from this workflow for this reason.

Reply:
But it could be more generic if it was in GvsUtils and only took the error message as a string argument.